scout/parse/panel.py

```diff
@@ -288,8 +292,19 @@ def parse_panel_app_gene(app_gene, hgnc_map, confidence):
     hgnc_symbol = app_gene["GeneSymbol"]
 
     ensembl_ids = app_gene["EnsembleGeneIds"]
+
+    if (
+        not ensembl_ids and hgnc_symbol in hgnc_symbol_ensembl_gene_map
```

Comment:
We haven't seen it yet, but since you do test, perhaps add a warning / error if not ensembl_ids and not in hgnc_symbol_ensembl_gene_map?

Reply:
Done